plugins/GSdx/GSState.h

```diff
@@ -218,6 +218,10 @@ class GSState : public GSAlignedClass<32>
 	GSPerfMon m_perfmon;
 	uint32 m_crc;
 	CRC::Game m_game;
+	// Needed for Automatic mipmapping.
+	// The original m_game variable is not used because then Automatic mipmapping will not work if CRC Hack Level is set to None.
+	// Using a separate variable addresses the mentioned issue.
+	CRC::Game m_game_mipmap;
```

Comment:
Maybe just use a local variable in GSRendererHW::SetGameCRC? It's not used anywhere else.